demo/src/docs/FullScreenLoader.js

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import PrismCode from "react-prism";
+import Typography from "elevate-ui/Typography";
+import withStyles from "elevate-ui/withStyles";
+import "../layout/prism-elevate.css";
+
+const FullScreenLoader = ({ classes }) => (
+  <div className={classes.wrapper}>
+    <div className={classes.headingContainer}>
+      <Typography
+        type="title"
+        gutterBottom
+      >{`<FullScreenLoader />`}</Typography>
+      <Typography type="body" className={classes.subheading}>
+        A component you can show for loading states.
+      </Typography>
+    </div>
+
+    <PrismCode component="pre" className="language-javascript">
+      {` 
+    import FullScreenLoader from 'elevate-ui/Loadable/FullscreenLoader';
```

Comment:
This was my typo from our slack convo. It should be  `FullscreenLoader` everywhere, although that goes against naming conventions. Maybe we should make sure `FullscreenLoader` is actually `FullScreenLoader` everywhere since that is the proper naming convention.

Reply:
Unless.... Well, it depends if you would say its Full Screen or Fullscreen. 🤔  either way, it needs to be consistent. 😂